src/app/clusters/messaging-client/messaging-client.c

```diff
@@ -31,18 +31,19 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-/***************************************************************************//**
- * @file
- * @brief Routines for the Messaging Client plugin, which implements the client
- *        side of the Messaging cluster.
- *******************************************************************************
-   ******************************************************************************/
+/***************************************************************************/ /**
```

Comment:
Probably better to insert a newline between those comments so this is formatted a bit more nicely.